src/components/Overview/PlotlyCharts/DataCollectedChart.tsx

```diff
@@ -7,58 +6,78 @@ import {
   AvastarParsedDataPointState,
 } from '../../../types/dataTypes';
 
+
 const Plot = createPlotlyComponent(Plotly);
+interface PropsFilter {
+  platform: string;
+  origin: string;
+  type: string;
+}
 
-export default function DataCollectedChart() {
-  // Fetch data from State
+export default function DataCollectedChart(props: PropsFilter) {
+  interface Anything {
+    [key: string]: any;
+  }
```

Comment:
Je ne comprends pas l'intérêt de créer une interface pour définir un type `any` ? 

A quoi correspond `Anything` ? Est-ce qu'on peut avoir un naming plus clair et/ou un commentaire pour comprendre l'objectif de cette variable ?

Reply:
J'ai mis un naming plus clair. Mais si tu as un moyen de pouvoir enlever la création de cette interface qui a pour seul objectif d'être utilisé en tant que définition de type vas-y! Si je ne fais pas ça j'ai des problèmes ts et je n'ai pas trouvé d'autres moyens pour m'en sortir.

Reply:
Migrated to dataTypes.ts